inferno/callbacks.py

```diff
@@ -402,55 +393,29 @@ class PrintLog(Callback):
       ("dur"). It will also highlight the best train and valid
       loss. If any of the mentioned keys is not found, it is ignored.
 
-    """
+      """
 
     default_keys = ['epoch', 'train_loss', 'valid_loss', 'train_loss_best',
                     'valid_loss_best', 'dur']
 
     def __init__(
             self,
-            keys=None,
-            keys_optional=None,
+            keys_ignored='batches',
             sink=print,
             tablefmt='simple',
             floatfmt='.4f',
     ):
-        if keys is None:
-            self.keys = []
-        else:
-            self.keys = [keys] if isinstance(keys, str) else keys
-
-        if keys_optional is None:
-            self.keys_optional = []
-        elif isinstance(keys_optional, str):
-            self.keys_optional = [keys_optional]
-        else:
-            self.keys_optional = keys_optional
-
+        if isinstance(keys_ignored, str):
+            keys_ignored = [keys_ignored]
+        self.keys_ignored = keys_ignored
```

Comment:
Maybe we should have something like 

    net.get_callback_by_name(name)

to modify this value later on?